src/firejail/main.c

```diff
@@ -1767,6 +1767,29 @@ int main(int argc, char **argv, char **envp) {
 			}
 		}
 #endif
+		else if (strncmp(argv[i], "--include=", 10) == 0) {
+			char *ppath = expand_macros(argv[i] + 10);
+			if (!ppath)
+				errExit("strdup");
+
+			char *ptr = ppath;
+			while (*ptr != '/' && *ptr != '\0')
+				ptr++;
+			if (*ptr == '\0') {
+				// ppath contains no '/', assume it's a name
```

Comment:
Would it be possible to somehow prioritize search in the current working directory?

Reply:
Done.

> Looks good to me. I still wonder if --include and --profile in the end should do the profile search the same way, including handling of file extensions.
>
>Then the only difference would be that --profile overrides automatic profile loading, whereas --include just loads additional instructions from a file.

I tried first, but I don't completely understand the `--profile` code. What does `has_colon`.